src/pages/MoviePage/MoviePage.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { Chip, Typography } from '@mui/material';
+import classNames from 'classnames';
+import style from './MoviePage.module.scss';
+import dataMovies from '../../mockedData/data-movies.json';
+import dataMovie from '../../mockedData/data-movie.json';
+import dataImages from '../../mockedData/data-images.json';
+import Card from '../../components/Card';
+import Paragraph from '../../components/Paragraph';
+import { TypeDataMovie } from '../../components/types';
+import ButtonElem from '../../components/ButtonElem';
+
+function MoviePage() {
+  const { original_title, poster_path, overview, release_date, revenue, runtime, genres } =
+    dataMovie as TypeDataMovie;
+  const [isOpenActors, setIsOpenActors] = useState(false);
+  const [textButton, setTextButton] = useState('Show all');
+  const actorsClassNames = classNames(style.actors__wrapper, {
+    [style.actors__active]: isOpenActors,
+  });
+
+  const isOpenActorsCollection = () => {
+    setIsOpenActors(!isOpenActors);
```

Comment:
If you want to change state based on the previous state, pass the callback instead of value.
setIsOpenActors((isOpenActors)=>!isOpenActors)